queries.sql

```diff
@@ -74,8 +74,35 @@ SELECT COUNT(*) FROM visits JOIN animals ON visits.animal_id = animals.id JOIN v
 
 SELECT species.name,COUNT(species.name) FROM visits JOIN animals ON visits.animal_id = animals.id JOIN vets ON visits.vet_id = vets.id JOIN species ON animals.species_id = species.id WHERE vets.name = 'Maisy Smith' GROUP BY species.name ORDER BY COUNT(species.name)DESC LIMIT 1;
 
+-- perfomance optimisation
 
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
 
 
+SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT COUNT(*) FROM  visits WHERE  animal_id = 4;
 
+SELECT * FROM visits where vet_id = 2;
+EXPLAIN ANALYZE SELECT  * FROM  visits WHERE vet_id = 2;
+
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+EXPLAIN ANALYZE SELECT  * FROM owners where  email = 'owner_18327@mail.com';
```

Comment:
- [ ] Kindly make the execution time for all queries before the optimization exceeds or equal `1000ms` as per this [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md#:~:text=4%20Depening%20on,bigger%20than%201000ms).

Reply:
Below are the s
![a-1](https://user-images.githubusercontent.com/71219070/208493915-f7378505-8452-46cb-93bd-00c18e4afbb6.PNG)
screenshots
![a-2](https://user-images.githubusercontent.com/71219070/208493979-83c9dbc7-3e3b-43d4-8248-aff8a9ba0354.PNG)
![a-3](https://user-images.githubusercontent.com/71219070/208494031-c4b77c45-5747-4906-b353-ff4506c523ec.PNG)
![a-4](https://user-images.githubusercontent.com/71219070/208494066-68c403fc-13be-43cb-b1d1-afa8fca1ef75.PNG)
![a-5](https://user-images.githubusercontent.com/71219070/208494112-f2680f4c-925c-4116-8c6a-df03c1740d54.PNG)
![a-6](https://user-images.githubusercontent.com/71219070/208494170-d7cb46f5-34f2-4f67-b480-2c80ba8c4acb.PNG)
 for the required changes